be/src/formats/parquet/column_converter.cpp

```diff
@@ -377,6 +385,11 @@ Status ColumnConverterFactory::create_converter(const ParquetField& field, const
             *converter = std::make_unique<PrimitiveToDecimalConverter<int32_t, TYPE_DECIMAL128>>(field.scale,
                                                                                                  typeDescriptor.scale);
             break;
+        case LogicalType::TYPE_TIME: {
```

Comment:
why put this logic in int32?
time type must be int64

Reply:
@Smith-Cruise  ALL DONE